Dockerfile

```diff
@@ -6,19 +6,23 @@ FROM triblercore/libtorrent:1.2.10-x
 RUN apt update && apt upgrade -y
 RUN apt install -y libsodium23 python3-pip git
 
+RUN useradd -ms /bin/bash user
```

Comment:
**question:** username "user" looks too general to me, can we use something more specific, like "dockeruser" or "docker"?

Reply:
It was intentional to make the user general and non-root as the user does not require any special rights